pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -8,14 +8,11 @@ class NumpyRecipe(CompiledComponentsPythonRecipe):
     version = '1.16.4'
     url = 'https://pypi.python.org/packages/source/n/numpy/numpy-{version}.zip'
     site_packages_name = 'numpy'
+    depends = ['setuptools']
 
     patches = [
         join('patches', 'add_libm_explicitly_to_build.patch'),
-        join('patches', 'do_not_use_system_libs.patch'),
         join('patches', 'remove_unittest_call.patch'),
-        join('patches', 'ar.patch'),
```

Comment:
Very nice to see that these can be removed, and it makes sense given the toolchain changes.